test/unit/org/authInfoTest.ts

```diff
@@ -245,6 +245,9 @@ describe('AuthInfo', () => {
     });
 
     it('should return an AuthInfo instance when passed a parent username', async () => {
+      stubMethod($$.SANDBOX, AuthInfo.prototype, 'determineIfDevHub').resolves(false);
+      stubMethod($$.SANDBOX, AuthInfo.prototype, 'getNamespacePrefix').resolves();
+
```

Comment:
if you checkout the commit before mine (`0f0d32fde65f893ff9fddd1169c8de750b575d51`) and run `JSFORCE_LOG_LEVEL=DEBUG yarn mocha test/unit/org/authInfoTest.ts` 

you'll see this:
```
➜  sfdx-core git:(dependabot-npm_and_yarn-jsforce-jsforce-node-3.2.0) ✗ JSFORCE_LOG_L
EVEL=DEBUG yarn mocha test/unit/org/authInfoTest.ts
yarn run v1.22.19
$ /Users/cdominguez/code/gh/sf/sfdx-core/node_modules/.bin/mocha test/unit/org/authIn
foTest.ts


  AuthInfo
    create
DEBUG   [fetch]  Request failed
DEBUG   [fetch]  retrying for the 1 time
DEBUG   [fetch]  Error: FetchError: request to https://instance.84edc3d01c22bbd5da27a
e8a686e2a52.salesforce.com//services/data/v51.0/query?q=SELECT%20Id%20FROM%20ScratchO
rgInfo%20limit%201 failed, reason: getaddrinfo ENOTFOUND instance.84edc3d01c22bbd5da2
7ae8a686e2a52.salesforce.com
DEBUG   [fetch]  Request failed
DEBUG   [fetch]  retrying for the 2 time
DEBUG   [fetch]  Error: FetchError: request to https://instance.84edc3d01c22bbd5da27a
e8a686e2a52.salesforce.com//services/data/v51.0/query?q=SELECT%20Id%20FROM%20ScratchO
rgInfo%20limit%201 failed, reason: getaddrinfo ENOTFOUND instance.84edc3d01c22bbd5da2
7ae8a686e2a52.salesforce.com
DEBUG   [fetch]  Request failed
DEBUG   [fetch]  retrying for the 3 time
DEBUG   [fetch]  Error: FetchError: request to https://instance.84edc3d01c22bbd5da27a
e8a686e2a52.salesforce.com//services/data/v51.0/query?q=SELECT%20Id%20FROM%20ScratchO
rgInfo%20limit%201 failed, reason: getaddrinfo ENOTFOUND instance.84edc3d01c22bbd5da2
7ae8a686e2a52.salesforce.com
```

jsforce v3.1 retried these failures very fast, v3.2 delays each retry incrementally so mocha just timed out.